sympy/polys/matrices/domainmatrix.py

```diff
@@ -49,21 +69,92 @@ def from_list_sympy(cls, nrows, ncols, rows, **kwargs):
 
     @classmethod
     def from_Matrix(cls, M, **kwargs):
+        """Converts a Matrix to Domain Matrix of
+        suitable Domain
+
+        Examples
+        ========
+        Define a Domain Matrix on the field of Rationals
+
+        >>> from sympy import Matrix, Rational
+        >>> from sympy.polys.matrices import DomainMatrix
+        >>> A = DomainMatrix.from_Matrix(Matrix([
+        ...     [Rational(1,2), Rational(3,4)],
+        ...     [2, Rational(1,5)],
+        ... ]))
+        >>> A
+        DomainMatrix([[1/2, 3/4], [2, 1/5]], (2, 2), QQ)
+
+        Define a Domain matrix on the field of Reals.
+        >>> from sympy import Matrix
+        >>> from sympy.polys.matrices import DomainMatrix
+        >>> A = DomainMatrix.from_Matrix(Matrix([
+        ...     [1.0, 3.4],
+        ...     [2.4, 1]
+        ... ]))
+        >>> A
+        DomainMatrix([[1.0, 3.4], [2.4, 1.0]], (2, 2), RR)
+
+        """
         return cls.from_list_sympy(*M.shape, M.tolist(), **kwargs)
 
     @classmethod
     def get_domain(cls, items_sympy, **kwargs):
+
         K, items_K = construct_domain(items_sympy, **kwargs)
         return K, items_K
 
     def convert_to(self, K):
+        """ Converts an object of a given Domain to another specified
+        domain K
+
+        Parameters
+        ==========
+
+        K: Matrix is to be converted to Domain K
+
+        Examples
+        ========
+
+        >>> from sympy.polys.matrices import DomainMatrix
+        >>> from sympy import QQ, ZZ
+        >>> A = DomainMatrix([[ZZ(1), ZZ(3)]], (1, 2), ZZ)
+        >>> A.convert_to(QQ)
+        DomainMatrix([[1, 3]], (1, 2), QQ)
+
+        """
         return self.from_rep(self.rep.convert_to(K))
 
     def to_field(self):
         K = self.domain.get_field()
         return self.convert_to(K)
 
     def unify(self, other):
+        """Unifies the domain of 2 Matrices, so that
+        they both belong to a common unified Domain
+
+        Parameters
+        ==========
+
+        other: other Domain Matrix to be unified
+
+        Returns
+        =======
+
+        Two Matrices that contain the elements of the input
+        matrices but they now belong to a unified Domain
+
+        Examples
+        ========
+
+        >>> from sympy.polys.matrices import DomainMatrix
+        >>> from sympy import ZZ, QQ
+        >>> A = DomainMatrix([[ZZ(1), ZZ(2), ZZ(3)]], (1, 3), ZZ)
+        >>> B = DomainMatrix([[QQ(1, 2), QQ(3, 5)]], (1, 2), QQ)
+        >>> A.unify(B)
+        (DomainMatrix([[1, 2, 3]], (1, 3), QQ), DomainMatrix([[1/2, 3/5]], (1, 2), QQ))
+
```

Comment:
This should have a "see also" for `convert_to` and also the `Domain.unify` method.